### PR TITLE
add styles

### PR DESCRIPTION
**Pull request by Braifz**
styles1
styles2
styles3

### CODE REVIEW

File: src/components/CardList.jsx
Comment (Barrod-code) [inline src/components/CardList.jsx:8]: 380px is too wide for smartphones, 320px is barely narrow enough to avoid side-scrolling. A better solution might be keeping 320p and cropping titles in order to avoid having irregular cards

File: .eslintcache
Comment (Barrod-code) [inline .eslintcache]: .eslintcache is already in .gitignore for this branch, I have no idea why this is still getting tracked.  @tonoezep any ideas?

Comment (tonoezep) [reply .eslintcache]: yes, Braifz has started this branch before this was added to gitignore, then merge it with the master's HEAD
just delete it, and start all new branches from the latest version of master ⭐

**Review by tonoezep — APPROVED** [review]

File: src/components/CardList.jsx
Comment (Braifz) [reply src/components/CardList.jsx:8]: Great! Now I make changes... what do you mean cropping titles? Less width?

Comment (Barrod-code) [reply src/components/CardList.jsx:8]: [Here](https://chakra-ui.com/docs/typography/text) in the Text docs, there is a isTruncated prop that should keep titles only 1 line long. This might do the trick

**Review by Barrod-code — APPROVED** [review]
Fix width of cards and then merge with squash. Click on the arrow to the right of merge button and you can select this option.

**Pull request merged**
